music21/meter/base.py

```diff
@@ -448,10 +448,62 @@ class TimeSignature(TimeSignatureBase):
     '2/4+3/16' or '11/16 (2/4+3/16)'.  Or you might want the written
     TimeSignature to contradict what the notes imply.  All this can be done
     with .displaySequence.
+
+    Equality
+    --------
```

Comment:
sadly -- it turns out this doesn't work -- Sphinx crashes.  Need to just label with bold: `**Equality**` and lose the heading marker `---------`.